keda/values.yaml

```
@@ -448,6 +448,14 @@ certificates:
     # has to be annotated with 'cert-manager.io/allow-direct-injection: "true"'
     generateCA: true
     caSecretName: "kedaorg-ca"
+    # Add labels/annotations to secrets created by Certificate resources
+    secretTemplate:
+      {}
```

Other cases are inline
```suggestion
    secretTemplate: {}
```